src/core/components/pin/gc-lock.js

```diff
@@ -0,0 +1,138 @@
+'use strict'
+
+const assert = require('assert')
+const mortice = require('mortice')
+const pull = require('pull-stream')
```

Comment:
```suggestion
const pull = require('pull-stream/pull')
```

https://www.npmjs.com/package/pull-stream#minimal-bundle